### PR TITLE
[ES] Update `Mascots`

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
- [x] #9565

### CODE REVIEW

File: wiki/Mascots/es.md [hunk 1]
```diff
@@ -70,36 +66,36 @@ Agent J también fue presentado en la skin de [LuigiHann](https://osu.ppy.sh/use
...
+Diseñada por [pyun](https://osu.ppy.sh/users/981534), Chirou fue una de las entradas en el concurso de diseño de mascotas para osu!taiko. Es muy estricta y exigente, perfeccionista y no le gusta cometer errores, especialmente en sus ritmos en osu!taiko. Pero más allá de su perspectiva fuerte, si llegas a su punto débil, puede ser esponjosa y linda. Tiene 14 años, nació el 25 de octubre con sangre tipo AB. Ella también mide 4 pies y 11 pulgadas y pesa 100 libras. Sus pasatiempos incluyen tocar la batería, mantenerse caliente en su capa y coleccionar gemas o rocas. Chirou ha hecho algunas apariciones en varias presentaciones de fanart de usuarios.
```
Comment (UnknownOsu) [inline wiki/Mascots/es.md:89]:
```suggestion
Diseñada por [pyun](https://osu.ppy.sh/users/981534), Chirou fue una de las candidatas en el concurso de diseño de mascotas para osu!taiko. Es muy estricta y exigente, perfeccionista y no le gusta cometer errores, especialmente en sus golpes en osu!taiko. Pero más allá de su perspectiva fuerte, si llegas a su punto débil, puede ser esponjosa y linda. Tiene 14 años, nació el 25 de octubre con sangre tipo AB. Ella también mide 4 pies y 11 pulgadas y pesa 100 libras. Sus pasatiempos incluyen tocar la batería, mantenerse caliente en su capa y coleccionar gemas o rocas. Chirou ha hecho algunas apariciones en varias presentaciones de fanart de usuarios.
```

File: wiki/Mascots/es.md [hunk 2]
```diff
@@ -70,36 +66,36 @@ Agent J también fue presentado en la skin de [LuigiHann](https://osu.ppy.sh/use
...
 
 ### Taikonator
 
 ![Taikonator](img/Taikonator.png "Taikonator")
 
-Taikonator, también conocido como Taikonator 3000, fue uno de los participantes en el concurso de diseño de mascotas para osu!taiko. Ganó atracción como un chiste y popularidad por razones desconocidas. Sus orígenes siguen siendo un misterio; a pesar de esto, es mucho más único que las otras sumisiones de mascotas para osu!taiko y ha aparecido en muchas sumisiones de fanart por usuarios de la comunidad.
+Diseñado por [Lemia-Chan](https://osu.ppy.sh/users/8506749), Taikonator, también conocido como Taikonator 3000, fue una de las entradas en el concurso de diseño de mascotas para osu!taiko. Ganó atracción como un chiste y popularidad por razones desconocidas. Sus orígenes siguen siendo un misterio; a pesar de esto, es mucho más único que las otras sumisiones de mascotas para osu!taiko y ha aparecido en muchas sumisiones de fanart por usuarios de la comunidad.
```
Comment (UnknownOsu) [inline wiki/Mascots/es.md:95]:
```suggestion
Diseñado por [Lemia-Chan](https://osu.ppy.sh/users/8506749), Taikonator, también conocido como Taikonator 3000, fue uno de los candidatos en el concurso de diseño de mascotas para osu!taiko. Ganó atracción como un chiste y popularidad por razones desconocidas. Sus orígenes siguen siendo un misterio; a pesar de esto, es mucho más único que los demás diseños de mascota para osu!taiko y ha aparecido muchas veces en fanarts de varios usuarios.
```

File: wiki/Mascots/es.md [hunk 3]
```diff
@@ -70,36 +66,36 @@ Agent J también fue presentado en la skin de [LuigiHann](https://osu.ppy.sh/use
...
 
 ### Tama
 
 ![Tama](img/Tama.png "Tama")
 
-Diseñada por [crystalsuicine](https://osu.ppy.sh/users/9974), Tama fue una de las participantes más jóvenes en el concurso de diseño de mascotas para osu!taiko con solo 15 años de edad, Tama tiene una gran pasión por tocar el tambor. Y tormentas eléctricas. Y especialmente los festivales, donde se arrebata cualquier takoyaki que se encuentra. Siempre lista para cualquier desafío, Tama oculta un pasado misterioso detrás de su comportamiento juvenil.
+Diseñada por [crystalsuicine](https://osu.ppy.sh/users/9974), Tama fue una de las entradas más jóvenes en el concurso de diseño de mascotas para osu!taiko con solo 15 años de edad y tiene una gran pasión por tocar el tambor. Y las tormentas eléctricas. Y especialmente los festivales, donde se arrebata cualquier takoyaki que se encuentra. Siempre lista para cualquier desafío, Tama oculta un pasado misterioso detrás de su comportamiento juvenil.
```
Comment (UnknownOsu) [inline wiki/Mascots/es.md:101]:
```suggestion
Diseñada por [crystalsuicine](https://osu.ppy.sh/users/9974), Tama fue una de las candidatas más jóvenes en el concurso de diseño de mascotas para osu!taiko con solo 15 años de edad y tiene una gran pasión por tocar el tambor. Y las tormentas eléctricas. Y especialmente los festivales, donde se arrebata cualquier takoyaki que se encuentra. Siempre lista para cualquier desafío, Tama oculta un pasado misterioso detrás de su comportamiento juvenil.
```

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

**Review by TicClick — APPROVED** [review]
merging (no native review for one week or longer)

**Review by TicClick — APPROVED** [review]

**Pull request merged**